win/win32service.cpp

```diff
@@ -232,7 +232,7 @@ void UninstallService()
 		CloseServiceHandle(SCMHandle);
 		std::cout << "Service removed." << std::endl;
 	}
-	catch(CWin32Exception e)
+	catch(CWin32Exception &e)
```

Comment:
Same as above.

Reply:
@SadieCat, fix it.
Commit info here:
https://github.com/inspircd/inspircd/pull/1879/commits/b4e0c5b6212def02386af0a555445412eed6f1f4